app.json

```diff
@@ -83,6 +83,9 @@
     },
     {
       "plan": "heroku-redis:hobby-dev"
+    },
+    {
+      "plan": "bonsai:sandbox-6"
```

Comment:
I'm a new to Heroku. For my information, how did you get the plan name "**sandbox-6**" ? I'm curious since I do not see it mentioned in the Bonsai documentation.

Reply:
I added it and then ran `heroku addons` from the Heroku CLI which lists all of the addons used.